test/fixtures/netweaver_simple_example/main.tf

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+resource "random_id" "random_suffix" {
+  byte_length = 4
+}
+
+locals {
+  gcs_bucket_name        = "post-deployment-bucket-${random_id.random_suffix.hex}"
+  gcs_bucket_static_name = "${var.nw_deployment_bucket}"
+}
+
+#TODO: Add creation of a network that's similar to app2
+
+resource "google_storage_bucket" "deployment_bucket" {
+  name          = "${local.gcs_bucket_name}"
+  force_destroy = true
+  location      = "${var.region}"
+  storage_class = "REGIONAL"
+  project       = "${var.project_id}"
+}
+
+data "template_file" "post_deployment_script" {
+  template = "${file("${path.cwd}/files/templates/post_deployment_script.tpl")}"
```

Comment:
We should consistently use `path.cwd` or `path.module`.

Reply:
used the path.module .